src/components/ComputeModal.js

```diff
@@ -624,6 +634,10 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
     }
   }
 
+  const copyImageLinkTooltipText = 'Copy the image version'
+  const isTerraSupported = runtime => (!runtime.isCommunity && !(getToolForImage(runtime.id) === tools.RStudio.label))
```

Comment:
```suggestion
  const isTerraSupported = ({ isCommunity, id }) => !isCommunity && !(getToolForImage(id) === tools.RStudio.label)
```
- Object deconstruction at input time makes it more clear what props (from `runtime` in this case) the function uses.
- The function body can often be written more concisely that way since `runtime` doesn't have to repeated.
- The outer parentheses should be redundant.

Reply:
I get this error when I try this formatting, any idea why?
![Screen Shot 2022-03-03 at 11 23 18 AM](https://user-images.githubusercontent.com/97535660/156606872-6b98f180-a48c-4e41-8b3c-2b9b20c7a679.png)

Reply:
Ah yea…`isCommunity` being `undefined` in the original version wasn't causing an error but it does in the proposed way. Assigning a default value should make it behave the same way it did before, and the intention would be more explicit:

```javascript
const isTerraSupported = ({ isCommunity = undefined, id }) => !isCommunity && !(getToolForImage(id) === tools.RStudio.label)
```